go.mod

```diff
@@ -65,7 +65,7 @@ require (
 	github.com/kevinburke/go-bindata v3.24.0+incompatible
 	github.com/kolide/launcher v1.0.12
 	github.com/lib/pq v1.10.9
-	github.com/macadmins/osquery-extension v1.0.1
+	github.com/macadmins/osquery-extension v1.1.2
```

Comment:
Can we just put `v1.1.3-0.20240530154548-05bb97403086` here?

Reply:
yup! TIL